content/python/concepts/built-in-functions/terms/endswith/endswith.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'endswith()'
+Description: 'Checks whether or not a string ends with the given character or string.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `endswith()` function in Python, takes in a string or character and checks whether or not the given string variable ends with it and the result is returned as a boolean value.
+
+## Syntax
+
+```py
+stringvariable.endswith("string")
+```
+
+## Example 1
+
+```py
+example= "This is a string"
+check = example.endswith("g")
+print(check)
+
+# Output: True
+```
+
+## Example 2
+
+```py
+example= "This is a string"
+check = example.endswith("s")
+print(check)
+
+# Output: False
+```
+
+## Example 3
+
```

Comment:
```suggestion

Checking for the value `ring`:
```

Reply:
@KTom101 Hey, just updated the file, hope it's fine now!